docs/providers/twitter.md

```diff
@@ -21,6 +21,10 @@ You can override any of the options to suit your own use case.
 
 ## Example
 
+:::note
+TWITTER_CLIENT_ID is the same as TWITTER_CONSUMER_KEY & TWITTER_CLIENT_SECRET is the same as TWITTER_CONSUMER_SECRET
```

Comment:
The environment variable name you choose doesn't really matter, so `TWITTER_CONSUMER_KEY` and `TWITTER_CONSUMER_SECRET` seem very arbitrary and I don't feel this explains anything better.

Instead of suggesting different environment variable names, I would explain what they represent, and link to the Twitter docs for explanation.

Reply:
@balazsorban44 I mean that's the term I've seen used in many tutorials online. It's never `CLIENT_ID` & `CLIENT_SECRET` but always `CONSUMER_KEY` & `CONSUMER_SECRET`. What do you think if I put this?

> API Key == Consumer Key. API Secret Key == Consumer Secret. See `Terminology clarification` section under https://developer.twitter.com/en/docs/authentication/oauth-1-0a/obtaining-user-access-tokens